.github/workflows/build.yml

```diff
@@ -209,6 +209,20 @@ on:
         description: |
           If remote store should be used, or instead remote builders.
 
+      eval_target:
+        type: string
+        default: ''
+        required: false
+        description: |
+          A single nix evaluation target that will be used to determine which outputs jobs should be created for. If an output is evaluated and determined to already be cached, it will not have a job created for it.
+
+      eval_top_attr:
+        type: string
+        default: ''
+        required: false
+        description: |
+          To avoid collisions with outputs that have the same `attr` but different `top_attr`, a top_atter for the eval target can be supplied here.
```

Comment:
```suggestion
          To avoid collisions with outputs that have the same `attr` but different `top_attr`, a top_attr for the eval target can be supplied here.
```